tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -140,7 +140,7 @@ virtiofsd-tarball:
 	${MAKE} $@-build
 
 merge-builds:
-	$(MK_DIR)/kata-deploy-merge-builds.sh build "$(MK_DIR)/../../../../versions.yaml"
+	$(MK_DIR)/kata-deploy-merge-builds.sh build "versions.yaml"
```

Comment:
This assumes that we "always" build at the top `kata-containers` directory. It fails e.g. for a build from the `local-build` directory. 
The issue is in `kata-merge-builds.sh` 
```
kata_versions_yaml_file_path="${PWD}/${kata_versions_yaml_file}"
````
This ^^ needs to be 
```
kata_versions_yaml_file_path="${kata_versions_yaml_file}"
```

Reply:
```
kata_versions_yaml_file_path="${PWD}/$(basename ${kata_versions_yaml_file})"
```

Reply:
${PWD} Will only work if you're in the `kata-containers` top directory.

Reply:
The `build` directory is "local" to your WD but not "versions.yaml" it can be "anywhere" depending on from where you start the build.

Reply:
Running at the top kata-containers directory: 

```

❯ pwd
/zvonkok/github.com/zvonkok/kata-containers


❯ make -f tools/packaging/kata-deploy/local-build/Makefile merge-builds                                              
echo /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build/
/zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build/
#/zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//kata-deploy-merge-builds.sh build /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../"versions.yaml"


❯ ls -l /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../"versions.yaml"
-rw-rw-r-- 1 zvonkok zvonkok 13795 Oct 12 10:15 /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../versions.yaml

```

Reply:
Running from `local-build` directory: 

```

❯ pwd
/zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build


❯ sudo make merge-builds                                               
echo /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build/
/zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build/
#/zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//kata-deploy-merge-builds.sh build /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../"versions.yaml"


❯ ls -l /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../"versions.yaml"
-rw-rw-r-- 1 zvonkok zvonkok 13795 Oct 12 10:15 /zvonkok/github.com/zvonkok/kata-containers/tools/packaging/kata-deploy/local-build//../../../../versions.yaml


```